src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -70,7 +70,12 @@ void setConversationStore(ConversationStore conversationStore) {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
-    List<Conversation> conversations = conversationStore.getAllConversations();
+    String username = (String) request.getSession().getAttribute("user");
```

Comment:
is casting as string necessary here?

Reply:
yes, it is necessary in java but not jsp.